src/lsp-server/src/backend.rs

```diff
@@ -16,15 +16,33 @@
 // The original source code is subject to the terms of the <APACHE|MIT> license, a copy
 // of which can be found in the LICENSE file at the root of this repository.
 
+use ::serde::Deserialize;
+use mz_ore::collections::HashMap;
+use mz_sql_parser::ast::{Raw, Statement};
 use regex::Regex;
 use ropey::Rope;
 use serde_json::Value;
+use tokio::sync::Mutex;
 use tower_lsp::jsonrpc::Result;
 use tower_lsp::lsp_types::*;
 use tower_lsp::{Client, LanguageServer};
 
 use crate::{PKG_NAME, PKG_VERSION};
 
+/// Default formatting width to use in the [Backend::formatting] implementation.
+pub const DEFAULT_FORMATTING_WIDTH: usize = 60;
```

Comment:
Bikeshed: 60 seems real short to me, but I'll defer to you. I use 60 on the website to generate interesting look demo output, but 100 is closer to what my eye prefers.